Header header
geometry_msgs/PoseWithCovariance[20] landmarkGTpose
int32[20] barcode

